daml-foundations/daml-ghc/tests/SubmitterInMaintainers_after_1866.daml

```diff
@@ -0,0 +1,88 @@
+-- @SINCE-LF 1.dev
```

Comment:
Can we please have a description of what we're testing here. Also, what is 1866? The module name should be in camel case. And we need a copyright header.

Reply:
Added a comment, and the copyright. Why the camelcase? I want to separate the test name and additional metadata (ticket number and whether it's before and after the fix)

Reply:
Oh, 1866 is a ticket number. This module is not named after the file. Also, `..._1866_before` and `..._after_1866` are not particularly consistent.